client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -1,39 +1,62 @@
 {{#let @form as |Form|}}
-  <Ui::Question class="fieldset relative scale-fade-in" as |Q|>
-    <span class="label">
-      {{~@form.saveableChanges.dcpZoningresolutiontype~}}
-      {{#if (lookup-action-type @form.saveableChanges.dcpZoningresolutiontype)}}
-        <LabsUi::IconTooltip
-          @tip={{lookup-action-type @form.saveableChanges.dcpZoningresolutiontype}}
-          @icon="info-circle"
-          @side="bottom"
-          @transform="shrink-4 up-2"
-          @fixedWidth={{true}}
-        />
+  <Ui::Question @fieldset={{true}} class="fieldset" as |Q|>
+    <Q.Prompt>
+      <span class="label">
+        {{~@form.saveableChanges.dcpZoningresolutiontype~}}
+        {{#if (lookup-action-type @form.saveableChanges.dcpZoningresolutiontype)}}
```

Comment:
We should be decisive about whether to use `@form` or the block param  (from `as |form|` ). If we introduce the block param, we don't need to use the argument `@form` here. Or, I think we can just get rid of `@form as |Form|` here, and use `@form` throughout.

Reply:
okay good point, I'll just use `@form as |Form|` and then use `form` throughout